packages/x-tree-view-pro/src/internals/plugins/useTreeViewItemsReordering/useTreeViewItemsReordering.itemPlugin.ts

```diff
@@ -36,9 +38,8 @@ export const useTreeViewItemsReorderingItemPlugin: TreeViewItemPlugin = ({ props
     itemId,
   );
   const isValidTarget = useSelector(store, selectorItemsReorderingIsValidTarget, itemId);
-  const draggedItemId = useSelector(store, selectorDraggedItem);
 
-  const isBeingEdited = useSelector(store, selectorIsItemBeingEdited, draggedItemId);
+  const isEditing = useSelector(store, selectorIsAnItemEdited);
```

Comment:
This will re-render all the items when starting editing.
In term of perfs, I think the best trade-off would be:

1. Use `!!isItemEditable` instead of `isEditing` on line 54 (if the label editing is not configured at all then we don't create the callbacks

2. Use the selector to know if an item is being edited inside `handleDragStart`, `handleRootDragOver` and `handleRootDragEnd`

The only diff is that `draggable` would be `true` even when editing an item, not sure if thats a problem.
If it is, then I don't have a great solution to avoid the full re-render :grimacing: